vmdb/app/models/hosts_storages.rb

```diff
@@ -1,3 +1,3 @@
 class HostsStorages < ActiveRecord::Base
-  table_name = "hosts_storages"
+  self.table_name = "hosts_storages"
```

Comment:
Yeah I think it was there originally for clarity because the model name doesn't match convention.